composer.json

```diff
@@ -1,54 +1,61 @@
 {
-  "name": "eightshift/eightshift-forms-plugin",
-  "version": "1.2.2",
-  "description": "This repository contains all the tools you need to start building a modern WordPress project.",
-  "keywords": [
-    "composer",
-    "installer",
-    "plugin",
-    "blocks",
-    "Gutenberg",
-    "WordPress"
-  ],
-  "homepage": "https://eightshift.com/",
-  "license": "MIT",
-  "authors": [
-    {
-      "name": "Eightshift team",
-      "email": "team@eightshift.com",
-      "homepage": "https://eightshift.com/",
-      "role": "Developer / IT Manager"
+    "name": "eightshift/eightshift-forms-plugin",
+    "version": "1.2.2",
+    "description": "This repository contains all the tools you need to start building a modern WordPress project.",
+    "keywords": [
+        "composer",
+        "installer",
+        "plugin",
+        "blocks",
+        "Gutenberg",
+        "WordPress"
+    ],
+    "homepage": "https://eightshift.com/",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Eightshift team",
+            "email": "team@eightshift.com",
+            "homepage": "https://eightshift.com/",
+            "role": "Developer / IT Manager"
+        }
+    ],
+    "support": {
+        "issues": "https://github.com/infinum/eightshift-forms-plugin/issues",
+        "source": "https://github.com/infinum/eightshift-forms-plugin"
+    },
+    "require-dev": {
+        "brain/monkey": "^2.5",
+        "codeception/codeception": "^4.1",
+        "codeception/module-asserts": "^1.0.0",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.7.0",
+        "infinum/coding-standards-wp": "0.4.3",
+        "php-stubs/wordpress-stubs": "^5.5"
+    },
+    "require": {
+        "php": ">=7.1",
+        "guzzlehttp/guzzle": "^6.0",
+        "infinum/eightshift-libs": "^2.2.2",
+        "saintsystems/odata-client": "^0.4.2"
+    },
+    "autoload": {
+        "classmap": [
+            "src/",
+            "tests/WordPressMocks/"
+        ],
+        "psr-4": {
+            "EightshiftFormsTests\\": "tests/"
+        }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "config": {
+        "sort-packages": true,
+        "optimize-autoloader": true,
+        "process-timeout": 2000
+    },
+    "scripts": {
+        "check-cs": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcs --runtime-set testVersion 7.1-",
+        "fix-cs": "@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf --runtime-set testVersion 7.1-"
```

Comment:
These should be set from the `phpcs.xml.dist`. Also, since you have tests, it's good to add additional scripts here as well 🙂 

I think I've added some in the testing with Codeception chapter in the handbook.

Reply:
Good idea, added just the unit one for now + phpstan.